.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2.1
 
 defaultImage: &defaultImage
-  image: "quay.io/rhacs-eng/apollo-ci:jenkins-plugin-0.3.27"
+  image: "quay.io/rhacs-eng/apollo-ci:snapshot-jenkins-plugin-0.3.28-74-g2d7d2e37a9" # TODO(do not merge): After upstream PR is merged, cut a tag and update this
```

Comment:
@vikin91 @gavin-stackrox  What's the status of this?

Reply:
I think this can be closed as out of date.